projects/shared/src/lib/view/brief/part-of/part-of.component.spec.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2021 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { TranslateModule } from '@ngx-translate/core';
+
+import { PartOfComponent } from './part-of.component';
+
+describe('PartOfComponent', () => {
+  let component: PartOfComponent;
+  let fixture: ComponentFixture<PartOfComponent>;
+
+  beforeEach(async () => {
```

Comment:
`async` is depracted, you should use `waitForAsync`

Reply:
That was generated automatically.